src/Goerli-DssSpell.t.sol

```diff
@@ -496,4 +497,320 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         assertEq(vest.rxd(1), WAD);
     }
+
+    function testNewEndAuthorities() public {
+        address endOld = chainLog.getAddress("MCD_END");
+        assertEq(vat.wards(endOld), 1);
+        assertEq(cat.wards(endOld), 1);
+        assertEq(dog.wards(endOld), 1);
+        assertEq(vow.wards(endOld), 1);
+        assertEq(pot.wards(endOld), 1);
+        assertEq(spotter.wards(endOld), 1);
+
+        assertEq(EndAbstract(endOld).wards(address(esm)), 1);
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // Contracts set
+        assertEq(end.vat(), address(vat));
+        assertEq(end.cat(), address(cat));
+        assertEq(end.dog(), address(dog));
+        assertEq(end.vow(), address(vow));
+        assertEq(end.pot(), address(pot));
+        assertEq(end.spot(), address(spotter));
+        assertEq(EndNewAbstract(address(end)).cure(), address(cure));
+
+        // Check end.wait
+        assertEq(end.wait(), EndAbstract(endOld).wait());
+
+        assertEq(esm.end(), address(end));
+
+        // Check flippers/clippers authorities and osms whitelisting
+        bytes32[] memory ilks = reg.list();
+        for (uint256 i = 0; i < ilks.length; i++) {
+            if (reg.class(ilks[i]) < 3) {
+                FlipAbstract xlip = FlipAbstract(reg.xlip(ilks[i]));
+                assertEq(xlip.wards(address(end)), 1);
+                assertEq(xlip.wards(endOld), 0);
+
+                OsmAbstract osm = OsmAbstract(reg.pip(ilks[i]));
+                try osm.bud(address(123)) { // Check is an OSM or Median
+                    assertEq(osm.bud(address(end)), 1);
+                    assertEq(osm.bud(endOld), 0);
+                } catch {}
+            }
+        }
+
+        assertEq(vat.wards(endOld), 0);
+        assertEq(cat.wards(endOld), 0);
+        assertEq(dog.wards(endOld), 0);
+        assertEq(vow.wards(endOld), 0);
+        assertEq(pot.wards(endOld), 0);
+        assertEq(spotter.wards(endOld), 0);
+
+        assertEq(EndAbstract(endOld).wards(address(esm)), 0);
+
+        assertEq(vat.wards(address(end)), 1);
+        assertEq(cat.wards(address(end)), 1);
+        assertEq(dog.wards(address(end)), 1);
+        assertEq(vow.wards(address(end)), 1);
+        assertEq(pot.wards(address(end)), 1);
+        assertEq(spotter.wards(address(end)), 1);
```

Comment:
For completeness we might also want to check `cure` here (I know we have a `cure` dedicated test but wouldn't hurt and this can be copied somewhere in the future).

Reply:
I see this is done now, so resolving.

Reply:
Ah yes forgot to comment on this one. I have also removed the specific test because it is actually pointless. I think for now we are fine just checking that the `End` process works. We can do a proper `Cure` test whenever we have a source and `wait` added.